### DISCUSSION TITLE
no minimize/close button

### DISCUSSION THREAD

**Discussion by therealgoodstuff** [FreeTubeApp/FreeTube#2937] [Q&A]
I have no minimize/make smaller/close buttons to the upper right anymore.
It looks like freetube entered a fullscreen mode (the video is not fs) and the buttons not visible/accessible anymore.
Happened with the new v0.18.0 Beta release.

Any ideas ?

cheers

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @therealgoodstuff what OS are u using?

**Reply by therealgoodstuff** [reply]
I use Windows 10 x64

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Did u try pressing F11?

**Reply by AdityaDas5501** [reply]
Worked! You saved my life!!
